Userland/Libraries/LibJS/Bytecode/Instruction.h

```diff
@@ -80,10 +80,12 @@ class Instruction {
 #undef __BYTECODE_OP
     };
 
+    bool is_terminator() const;
     Type type() const { return m_type; }
     size_t length() const;
     String to_string(Bytecode::Executable const&) const;
     void execute(Bytecode::Interpreter&) const;
+    void replace_references_to(BasicBlock const&, BasicBlock const&);
```

Comment:
This declaration is highly ambiguous.
The `_to` suffix in the name makes it sound like the first parameter is the "to" target, which it is not.
The parameters are not named and both have the same type so it's entirely unclear which is which.
How about:
```cpp
void replace_block_references(BasicBlock const& from, BasicBlock const& to);
```

Reply:
I mean... We'll also need to replace uses of registers, which is why I kept the name generic (so we could later add a Register overload)
Would removing the `_to` resolve this?

Reply:
Oh right, now I see why it was named `replace_references_to`.
classes inherit the function named `replace_references` from `Instruction`, so not implementing it wouldn't be a compile error, just a runtime infinite recursion...

Reply:
I believe the same can happen for `execute()`.